DarkMode Feature Updated

Rajdip-Pal opened the pull request:
# [Feature] - DARK Mode

@sitambhra 
I have added the dark mode feature as per assigned to me in the issue #5 

Here's the results

![Screenshot from 2025-02-23 03-05-18](https://github.com/user-attachments/assets/c0f3a821-e384-4992-9b3f-c36327e9d405)

DEMO deployment Link : https://reeltor-eosin.vercel.app/

Rajdip-Pal committed:
DarkMode Feature Updated

Rajdip-Pal commented:
@sitambhra 
Will you please reply me 
do you have any problem with The current PR if any issues occured with the PR you can tell me I will try my best to resolve it.

sitambhra commented:
@Rajdip-Pal 
Apologies for the late reply!

I see few issues and recommend you to fix them:

1. In dark mode, the notification bell is not visible, changes its color from black to white for dark mode only.
![image](https://github.com/user-attachments/assets/f19b40a5-d50a-4f49-8ea9-6c26e24fbd91)
2. If you see closely, both the icons are not horizontally aligned in a straight line
![image](https://github.com/user-attachments/assets/99219212-abd4-493a-9956-c71b60798af5)
3. When i over moon icon, change the cursor to pointer.

Also, i can see that you are processing project with nelify, i recommend using vercel since original project is deployed on verecl only.

Rajdip-Pal committed:
Fixing issues and adding demanded features

Rajdip-Pal commented:
> @Rajdip-Pal Apologies for the late reply!
> 
> I see few issues and recommend you to fix them:
> 
> 1. In dark mode, the notification bell is not visible, changes its color from black to white for dark mode only.
>    ![image](https://private-user-images.githubusercontent.com/110994217/415995760-f19b40a5-d50a-4f49-8ea9-6c26e24fbd91.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA0MDc1NzgsIm5iZiI6MTc0MDQwNzI3OCwicGF0aCI6Ii8xMTA5OTQyMTcvNDE1OTk1NzYwLWYxOWI0MGE1LWQ1MGEtNGY0OS04ZWE5LTZjMjZlMjRmYmQ5MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIyNFQxNDI3NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hODZjY2RlMTI2NTNkMTUxMjhjMTA1NjYwNjdjNzQwMjQxODIwMDhiYjUxZWNmMDAyYjlmNDYzODQwNDY0OWI4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.fgpQh8_a52Aw59HldIqWGq19_RXO_yVV5rHmuoi7df0)
> 2. If you see closely, both the icons are not horizontally aligned in a straight line
>    ![image](https://private-user-images.githubusercontent.com/110994217/415995885-99219212-abd4-493a-9956-c71b60798af5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA0MDc1NzgsIm5iZiI6MTc0MDQwNzI3OCwicGF0aCI6Ii8xMTA5OTQyMTcvNDE1OTk1ODg1LTk5MjE5MjEyLWFiZDQtNDkzYS05OTU2LWM3MWI2MDc5OGFmNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIyNFQxNDI3NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04MWM3ZmMxYjI1NmY4ZTdhNjdlYjhmMGU5ZjJlYTkyZWE0N2JkMzkzODkxOTkxNmE4N2JhMGE0MzI3NWUzNDYzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.1zs9ymV4Tq45XjKcQV1eipEXId197MAeYS0ynmcFuMU)
> 3. When i over moon icon, change the cursor to pointer.
> 
> Also, i can see that you are processing project with nelify, i recommend using vercel since original project is deployed on verecl only.

# Issues Fixed

@sitambhra 
I fixed the issues you asked for and also added the features you are additionally asking for 

## Fixed issues : 

### 1. Alignments of icons : 
![image](https://github.com/user-attachments/assets/9b365766-186b-4054-975f-4cdd51676994)

I guess the icons are aligned properly now which is giving the navbar a good look.

### 2. Notification icon visibility :
![image](https://github.com/user-attachments/assets/1243f969-4d08-43ef-827b-4c30209d7aca)
![image](https://github.com/user-attachments/assets/ec65cc00-99d9-4d02-88e8-076954e80de1)


Currently the notification icon also have the theme changine effect.

### 3. Pointer Effect on hover : 

I also added the pointer effect on hovering which is visible in website also.

I guess i fixes all the issues you mentioned for , 
here you can check the demo deployment : https://reeltor-eosin.vercel.app/
Hope this will do it, If any problem occurs please tell me I will try my best to resolve it.

sitambhra commented:
@Rajdip-Pal 
![image](https://github.com/user-attachments/assets/63bd2d31-4293-4c7b-b14d-51bcec2f544c)
In dark mode, this "<" symbol is not visible. kindly ensure the visibility

Rajdip-Pal committed:
Fixing issues and adding demanded features 2.0

Rajdip-Pal commented:
@sitambhra You can check it now .

sitambhra commented:
@Rajdip-Pal 
Good work!
Keep it up
